package.json

```diff
@@ -11,7 +11,9 @@
   },
   "scripts": {
     "start": "concurrently \"yarn server\" \"yarn client\"",
+    "start:prod": "concurrently \"yarn server\" \"yarn client:prod\"",
```

Comment:
start in prduction to test in production before finalising the deployment process